src/pybind/mgr/volumes/fs/operations/versions/subvolume_base.py

```diff
@@ -13,6 +14,11 @@
 
 log = logging.getLogger(__name__)
 
+@unique
```

Comment:
I suggest not using enum module for the sake of compatibility with earlier versions of python, maybe this can just be,
```
class SubvolumeFeatures():
    FEATURE_SNAPSHOT_CLONE = "snapshot-clone"
    FEATURE_SNAPSHOT_AUTOPROTECT = "snapshot-autoprotect"
```